### ISSUE TITLE
Workflow tests needed to avoid critical breakage on live system

### ISSUE THREAD

**Issue by baob** [swapmyvote/swapmyvote#975]
Examples

* User onboard from home page (ref bug #824, see also #922)
* user onboard via registration page (accessed via login page)
* User onboard via "API"

* Swap initiation with mobile set and verified
* swap initiation with mobile set and not verified (see #974) 

* password reset (email method has bugs)

*  email validation (feature doesn't exist right now and should, see #15)

**Comment by aspiers** [comment]
Yes agreed :+1:  Another key thing is that when changing partials we need to check the codebase for all the flows which use them, not just the ones we can find / remember via UI.

**Comment by aspiers** [comment]
Really we need https://www.cypress.io/ or similar for E2E tests.
